src/plugin.ts

```diff
@@ -139,12 +139,11 @@ export default class Plugin {
   validateFunctions(instance: ConcurrencyFunction): boolean {
     return !!(
       instance.provisionedConcurrency &&
-      instance.provisionedConcurrency > 0 &&
```

Comment:
@claydanford can you give me some insight on why we check that provisionConcurrency is there and that it is greater than 0?

Reply:
@nmrad-91 If `provisionedConcurrency` is not there then we cannot add concurrency autoscaling. If `provisionedConcurrency` is not >0 then we cannot add concurrency autoscaling.